<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CModelGraph1Widget</class>
 <widget class="QWidget" name="CModelGraph1Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>331</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>横坐标：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditX"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>纵坐标：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditY"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>颜色：  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineRGB">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnColor">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>数据源：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="txtDataSrc"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
